tmva/tmva/inc/TMVA/TSpline1.h

```diff
@@ -53,11 +52,10 @@ namespace TMVA {
       virtual void BuildCoeff( void );
       virtual void GetKnot( Int_t i, Double_t& x, Double_t& y ) const;
 
-      const TGraph* GetGraph() const { return fGraph; }
-
    private:
-
-      TGraph *fGraph;  ///< graph that is splined
+      int N;
+      std::vector<Double_t> X;
```

Comment:
It would be better to follow the ROOT coding convention and have data members starting with the letter `f`.
So can you please rename `X` -> `fX`  and `Y` -> `fY` ?